docs/backend/guides/service-objects/overview.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 1
+---
+# Overview
+
+## What is a Service Object?
+
+The MilMove backend is _loosely_ designed with this
+[3-layer structure](https://docs.google.com/presentation/d/1kVQzrYWY0AnYyPbiqfuFv8Fh_7IwwIFv3XKRxZI44Hs/edit#slide=id.p):
+
+1. **Presentation layer.** - Our endpoint handlers that perform type conversions and connect user requests to 
+   business logic functions.
+2. **Business logic layer.** - Code that implements MilMove's business logic.
+3. **Data access layer.** - Code that directly interacts with and manipulates the database.
+
+The `services` package is a combination of the bottom two layers, **business logic** and **data access**. It is 
+located within the [`mymove` `./pkg/services`](https://github.com/transcom/mymove/tree/master/pkg/services) directory.
+
+Our "service objects," as we call them (this is MilMove-specific terminology), are the structs/functions within this 
+package that implement our business logic. We have a few different kinds of service objects:
```

Comment:
Since the person reading "What is a service object" might literally have no clue what one is, I think it would be good to say "like AddressUpdater, MoveCreator". Otherwise this is very abstract and hard to connect to specific code.

Reply:
Good point, I'll add this!

Reply:
I've updated the docs to have examples like this. Lmk if it looks better, or if you think more is needed. Relevant commit: https://github.com/transcom/mymove-docs/pull/137/commits/b0e1d7362b02f3ed778cee1272e93140ecfbd5ee